feature-libs/quote/core/connectors/quote.adapter.ts

```diff
@@ -0,0 +1,110 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { PaginationModel } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import {
+  QuoteList,
+  QuoteStarter,
+  Quote,
+  QuoteMetadata,
+  QuoteActionType,
+  Comment,
+  QuoteDiscount,
+} from '@spartacus/quote/root';
+
+export abstract class QuoteAdapter {
+  /**
+   * Abstract method used to list of quotes for user
```

Comment:
Should it be mentioned by every methods that a method is an abstract one?

Reply:
@ChristophHi, what do you think?

Reply:
I guess not. Will change that